libs/model/src/models/types.ts

```diff
@@ -6,6 +6,39 @@ import {
   type BuildOptions,
 } from 'sequelize';
 
+/**
+ * Association rule options
+ * - `onUpdate`: update rule, 'NO ACTION' by default
+ * - `onDelete`: delete rule, 'NO ACTION' by default
+ */
+export type RuleOptions = {
+  onUpdate?: 'CASCADE' | 'NO ACTION' | 'SET NULL';
```

Comment:
Can we import the type from Sequelize instead of hardcoding it?

Reply:
will do, I also need to solve new conflicts with the evm outbox

Reply:
@timolegros where can I find those types in sequelize btw?

Reply:
Not sure I was just hoping we could extract them. Feel free to resolve if we can't import.

Reply:
can't find them...resolving this